subcommittees/maintainers/release-process.md

```diff
@@ -0,0 +1,50 @@
+# Release Process and Schedule
+
+We have decided to use a **6-month release cycle** for releases, which
+will be named by the year and month they happen, e.g., `2016.05`.
+
+## Overview
+
+1. Each lesson will continue live in the `gh-pages` branch of its own repository.
+2. When a release has to be made, the *lesson maintainer* (or maintainers) will create a branch named after the release, e.g., `2016.05`.
+3. A *release maintainer* will generate HTML pages for that release and add them to the branch.
```

Comment:
I think that's a separate issue from how we flag particular releases - 
want me to bring it to the maintainers?